dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,33 +1,45 @@
 -# English only for now
 - if language == "en"
 
-  -# Never show the survey.
-  - if false
+  -# US only
+  - if request.location.try(:country_code) == 'US' || (!Rails.env.production? && request.location.try(:country_code) == 'RD')
 
     -# If signed in, and user hasn't already filled out this survey
-    - if current_user && !current_user.survey2015_value
+    - if current_user && SurveyResult.where(user_id: current_user.id).blank?
```

Comment:
You can use exists here: http://apidock.com/rails/ActiveRecord/FinderMethods/exists%3F

In this case the difference is not a big deal but in general it will save you from transferring the row and allocating a SurveyResult object